### DISCUSSION TITLE
change node.name without restart

### DISCUSSION THREAD

**Discussion by bahientn9x** [emqx/emqx#8621] [Q&A] [answered]
My node currently has 50,000 clients connecting to it. I want to run bridging or cluster but need to name the node. But a reboot is required to confirm the new node name. Restart will cause the clients to lose connection and I don't want that to happen. Any ideas for this?
Thanks.

**Comment by zmstone** [comment, marked as answer]
Hi @bahientn9x 
I'm afraid renaming a node on the fly is not possible.
